ployst/github/hooker.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+import hashlib
+import hmac
+import json
+
+
+DEBUG = True
+
+
+class GithubHookException(Exception):
+    pass
+
+
+def compute_github_signature(body, secret):
+    return hmac.new(secret, body, hashlib.sha1).hexdigest()
+
+
+class GithubHookHandler(object):
+    """
+    A general purpose-class to help build github hook handlers.
+
+    See https://developer.github.com/webhooks/ for Github's webhook spec.
+
+    NOTE: This module may be spin-off into a separate library.
+    """
+
+    @staticmethod
+    def get_org_repo(request):
```

Comment:
Isn't the reason for this secret being unique so that you can't fake requests to the API.
If we had one globally used one, you would only need to sign up for your repo to then use it to make subsequent requests pretending to be from github.

To solve your use case too, perhaps we should only use the organization name to generate the secret, rather than repo too?